pkg/mimirtool/commands/analyse_grafana.go

```diff
@@ -47,7 +65,13 @@ func (cmd *GrafanaAnalyzeCommand) run(k *kingpin.ParseContext) error {
 		return err
 	}
 
+	filterOnFolders := len(cmd.folders) > 0
+
 	for _, link := range boardLinks {
+		if filterOnFolders && !slices.Contains(cmd.folders, link.FolderTitle) {
+			fmt.Fprintf(os.Stderr, "%s\n", link.FolderTitle)
```

Comment:
Is it useful to print just the folder title out here, should the message include some extra context?

Reply:
not at all, this should be remove.